File: test/export/php/02-entrada-salida.php
<?php
	$stdin = fopen('php://stdin','r');
	settype($a,'integer');
	settype($b,'float');
	settype($c,'string');
	settype($d,'boolean');
	fscanf($stdin,"%d",$a);
	fscanf($stdin,"%f",$b);
	$c=rtrim(fgets($stdin),PHP_EOL);
	fscanf($stdin,"%d",$d);
	echo $a,PHP_EOL;
	echo $b,PHP_EOL;
	echo $c,PHP_EOL;
	echo $d,PHP_EOL;
	fscanf($stdin,"%d",$a);
	fscanf($stdin,"%f",$b);
	$c=rtrim(fgets($stdin),PHP_EOL);
	fscanf($stdin,"%d",$d);
	echo $a,$b,$c,$d,PHP_EOL;
	echo 'Esta linea no lleva enter al final';
?>

File: test/export/c/02-entrada-salida.c
#include<stdio.h>
#include<stdbool.h>
#define MAX_STRLEN 256
int main() {
	int a;
	float b;
	char c[MAX_STRLEN];
	bool d;
	scanf("%i",&a);
	scanf("%f",&b);
	scanf("%s",c);
	scanf("%i",&d);
	printf("%i\n",a);
	printf("%f\n",b);
	printf("%s\n",c);
	printf("%i\n",d);
	scanf("%i",&a);
	scanf("%f",&b);
	scanf("%s",c);
	scanf("%i",&d);
	printf("%i%f%s%i\n",a,b,c,d);
	printf("Esta linea no lleva enter al final");
	return 0;
}
